### PR TITLE
Create an ADR to describe the InVEST LS Factor update

### LINKED ISSUE #1079: Proposal: store technical documents in InVEST
## Background
This is a co-sponsored proposal between @dcdenu4 and @phargogh that came from a chat about whether the SDR LS factor rework warranted a PEP.

## Problem
Given the scientific nature of our software there are times when we need to update an equation, rework an algorithm, or make some kind of non-trivial change to a model. These changes often don't require full PEP write ups and the discussion and reasoning behind the changes don't fully get captured in related issues, PR descriptions, history notes, or the Users Guide. This often leads to that historical knowledge being confined to the modeler and software engineer who made the changes. This causes future us to try and decipher why certain changes were made and what the reasoning behind them was.

## Suggested solution
When a non-trivial InVEST model change is needed, a write up of that change and decision should be stored in the InVEST repository so that it can be referenced in the future. GDAL does [something similar](https://github.com/OSGeo/gdal/tree/master/doc/source/development/rfc) with a `doc/source/development/rfc` structure that we could take inspiration from.

## Scope of work
This is proposal is for adopting a new software team practice and does not require any development time.

### PR DESCRIPTION
**Pull request by phargogh**
This PR implements an ADR for the upcoming changes to the InVEST LS Factor.

@dcdenu4 and I were talking and since this is our first ADR to be submitted, it'd be great to hear your thoughts (@davemfish @emlys ), particularly on the structure of the files and content of the ADR, but of course any feedback on the ADR itself is welcome!

Fixes #1079 

## Checklist
- [x] Updated HISTORY.rst and link to any relevant issue (if these changes are user-facing)
~~- [ ] Updated the user's guide (if needed)~~
~~- [ ] Tested the affected models' UIs (if relevant)~~

### CODE REVIEW

File: doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+was much more realistic.
+
+After diving into the SAGA source code, it turns out that there are several
+important differences between the two despite both using Desmet & Govers (1996)
+for their LS factor equations:
+
+1. The contributing area $A_{i,j-in}$ is not strictly defined in Desmet &
+   Govers (1996), it is only referred to as "the contributing area at the inlet
+   of a grid cell with coordinates (i, j) (m^2)".
+   InVEST assumes that "contributing area" is $area_{pixel} \cdot n\_upstream\_pixels$.
```
Comment (dcdenu4) [inline doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md:33]: On inspecting the rich diff, it looks like there might be some syntax issues around subscripting. I noticed a few subscripts that didn't look as expected.

**Review by dcdenu4 — CHANGES_REQUESTED** [review]
Thanks @phargogh, this looks awesome and I think will be super useful. Just a minor comment about some Markdown subscripts.

File: doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# ADR-0001: Update the InVEST SDR LS Factor
+
+Author: James
```
Comment (davemfish) [inline doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md:3]: need some trailing spaces for newline here

File: doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Once implemented and released, the LS factor outputs of SDR will be
+significantly different, but they should more closely match reality.
+
+We hope that there will be fewer support requests about this once the change is
+released.
+
+## References
+
+Zevenbergen & Thorne (1987): https://searchworks.stanford.edu/articles/edb__89861226
```
Comment (davemfish) [inline doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md:87]: need trailing spaces or bullets, etc, for newlines

**Review by davemfish — CHANGES_REQUESTED** [review]
This looks like a very useful document! I like how it lists out the history of relevant changes.

File: doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md (hunk 2 above)
Comment (phargogh) [reply doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md:3]: Patched in [8e2b41e](https://github.com/natcap/invest/pull/1288/commits/8e2b41e8542ac24f69092f32a30323c3e7f92f61)

File: doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md (hunk 3 above)
Comment (phargogh) [reply doc/decision-records/ADR-0001-Update-SDR-LS-Factor.md:90]: Patched in [8e2b41e](https://github.com/natcap/invest/pull/1288/commits/8e2b41e8542ac24f69092f32a30323c3e7f92f61)

**Review by davemfish — APPROVED** [review]

**Review by emlys — APPROVED** [review]
Great summary of the change!

**Pull request merged**